docs/ssdt/download-sql-server-data-tools-ssdt.md

```diff
@@ -56,7 +56,7 @@ For Analysis Services, Integration Services, or Reporting Services projects, you
 
 | Project Templates | SQL Platforms Supported |
 | --- | --- |
-| Relational databases | [!INCLUDE [sssql16-md](../includes/sssql16-md.md)] - [!INCLUDE [sssql22-md](../includes/sssql22-md.md)]<br /><br />Azure SQL Database, Azure SQL Managed Instance<br /><br />Azure Synapse Analytics (dedicated pools only) |
+| Relational databases | [!INCLUDE [sssql16-md](../includes/sssql16-md.md)] - [!INCLUDE [sssql22-md](../includes/sssql22-md.md)]<br /><br />Azure SQL Database, Azure SQL Managed Instance<br /><br />Azure Synapse Analytics Dedicated Pools <br /> Azure Synapse Analytics Serverless Pools (requires VS2022 17.7 see [release Notes](https://learn.microsoft.com/en-us/visualstudio/releases/2022/release-notes-v17.7#support-for-serverless-sql-pool-in-ssdt)) |
```

Comment:
looks great, thank you for this!
once you remove the en-us from `https://learn.microsoft.com/en-us/visualstudio/releases/2022/release-notes-v17.7#support-for-serverless-sql-pool-in-ssdt` I can sign off.

the url https://learn.microsoft.com/visualstudio/releases/2022/release-notes-v17.7#support-for-serverless-sql-pool-in-ssdt will auto-resolve to a user's docs language